src/pages/view/FilterBar/styles/FilterBadge.module.css

```diff
@@ -22,12 +22,29 @@
 }
 
 .removeButton {
+    background-color: transparent;
+    color: white;
+    border: none;
+    padding: 0;
     font-size: 12px;
     text-decoration: underline;
     margin-left: var(--spacers-dp24);
     cursor: pointer;
 }
 
+.span {
+    display: inline-flex;
+    pointer-events: all;
+}
+
+.span > :global(button:disabled) {
```

Comment:
what does `:global` do here?

Reply:
I'm not sure, this was the css Hendrick gave me to resolve the issue of a disabled button that can have a tooltip.